.chloggen/codeboten_add-release-opampsupervisor.yaml

```
@@ -0,0 +1,26 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: opampsupervisor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: publish a container for the opampsupervisor binary
```

```suggestion
note: publish binaries and container images for the OpAMP Supervisor
```
This publishes binaries as well, right?

---

yup